src/schemas/TopicsSchema.js

```diff
@@ -0,0 +1,19 @@
+var graphql = require('graphql');
+ 
+module.exports = graphql.buildSchema(`
+
+  type Topic {
+    id: Int!,
+    topic: String!,
+    value: String!
+  }
+
+  type TopicCollection {
+    collection: [Topic]!
+  }
+  
+  type Query {
+    get(containerName: String!, blobName: String!, id: Int!): Topic
```

Comment:
Yes, good point. I agree, if we move away from blob storage, we won't really care about the container name. Say we get info from Cassandra, we would just need the tablename and id. I will update this query to reflect that.